projects/observability/src/pages/saved-queries/saved-queries.component.ts

```diff
@@ -0,0 +1,51 @@
+import { ChangeDetectionStrategy, Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+
+import { NavigationParamsType, NavigationService, PreferenceService } from '@hypertrace/common';
+import { SavedQuery } from '../explorer/explorer.component';
+
+@Component({
+  styleUrls: ['./saved-queries.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="saved-queries">
+      <ht-page-header class="explorer-header"></ht-page-header>
+      <div class="query-list-container">
+        <a *ngFor="let query of savedQueries" (click)="onClick(query)">
+          <span *ngFor="let label of query.labels">{{ label }}</span>
+        </a>
+      </div>
+    </div>
+  `
+})
+export class SavedQueriesComponent implements OnDestroy {
+  public savedQueries: SavedQuery[] = [];
+  private readonly subscriptions: Subscription = new Subscription();
+
+  public constructor(
+    private readonly navigationService: NavigationService,
+    private readonly preferenceService: PreferenceService
+  ) {
+    this.subscriptions.add(
+      this.preferenceService.get('savedQueries', []).subscribe(queries => {
+        this.savedQueries = queries as SavedQuery[];
+      })
+    );
+  }
+
+  public ngOnDestroy(): void {
+    this.subscriptions.unsubscribe();
+  }
+
+  public onClick(query: SavedQuery): void {
+    this.navigationService.navigate({
+      navType: NavigationParamsType.InApp,
+      path: ['/explorer'],
+      queryParams: {
+        filter: query.filters,
+        scope: query.scope
+      },
+      replaceCurrentHistory: true
+    });
```

Comment:
Alternate approach: Instead of providing all these nav params separately, we can use the `buildNavParamsWithFilters` method in `ExplorerService`.

To use that, we'll need to save the full queries instead of just the scope and urlStrings.
cc: @jaywalker21